distributed/cli/dask_remote.py

```diff
@@ -9,8 +9,12 @@
 @click.option('--host', type=str, default=None,
               help="IP or hostname of this server")
 @click.option('--port', type=int, default=8788, help="Remote Client Port")
-def main(host, port):
-    _remote(host, port)
+@click.option('--certfile', default=None,
```

Comment:
These options (and the corresponding logic) is duplicated several times, maybe it would be good to factor them out.